packages/transformers/babel/src/config.js

```diff
@@ -53,6 +53,7 @@ export async function load(
     cwd: path.dirname(config.searchPath),
     root: options.projectRoot,
     babelrcRoots,
+    envName: options.env.BABEL_ENV || options.env.NODE_ENV || 'development',
```

Comment:
I think `options.mode` should be factored into the fallback?

Reply:
In the cli, we set a `NODE_ENV` if one isn't provided depending on if Parcel is invoked with `build`, but this could certainly apply when Parcel is invoked via the js API.